tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -132,6 +134,11 @@ def test_pipelining(self):
             False, forward_backward_pipelining_without_interleaving, None, None
         )
 
+    def test_pipelining_async(self):
```

Comment:
what do you think about adding `test_pipelining_async_inference` before merging?